### PR TITLE
feat: IPs and port mappings in pod get output

### PR DESCRIPTION
**Pull request by pw-git**
closes runpod/runpodctl#39

### CODE REVIEW

File: cmd/pod/getPod.go [hunk 1]
```diff
@@ -30,6 +30,20 @@ var GetPodCmd = &cobra.Command{
 			}
 			row := []string{p.Id, p.Name, fmt.Sprintf("%d %s", p.GpuCount, p.Machine.GpuDisplayName), p.ImageName, p.DesiredStatus}
 			if AllFields {
+
+				var portEntries int = 0
+				if  p.Runtime != nil && p.Runtime.Ports != nil {
+					portEntries = len(p.Runtime.Ports)
+				}
+				ports := make([]string, portEntries)
+				for j:=0; j<portEntries; j++ {
```
Comment (ef0xa) [inline cmd/pod/getPod.go:39]: for j := range ports {

}

File: cmd/pod/getPod.go [hunk 2]
```diff
@@ -38,14 +52,15 @@ var GetPodCmd = &cobra.Command{
 					fmt.Sprintf("%d", p.ContainerDiskInGb),
 					fmt.Sprintf("%d", p.VolumeInGb),
 					fmt.Sprintf("%.3f", p.CostPerHr),
+					fmt.Sprintf("%s", strings.Join(ports[:], "\n") ),
```
Comment (ef0xa) [inline cmd/pod/getPod.go:55]: are these supposed to be separated by newlines?

File: cmd/pod/getPod.go [hunk 3]
```diff
@@ -30,6 +30,20 @@ var GetPodCmd = &cobra.Command{
...
+				if  p.Runtime != nil && p.Runtime.Ports != nil {
+					portEntries = len(p.Runtime.Ports)
+				}
+				ports := make([]string, portEntries)
+				for j:=0; j<portEntries; j++ {
+					var privpub string = "prv"
+					if p.Runtime.Ports[j].IsIpPublic {
+						privpub = "pub"
+					}
+					ports[j] = fmt.Sprintf("%s:%d->%d\u00A0(%s,%s)",  p.Runtime.Ports[j].Ip,  p.Runtime.Ports[j].PublicPort,  p.Runtime.Ports[j].PrivatePort,  privpub,  p.Runtime.Ports[j].PortType)
```
Comment (ef0xa) [inline cmd/pod/getPod.go:44]: I am deeply suspicious of `0xA0`, the non-breaking space.

Comment (DireLines) [reply cmd/pod/getPod.go]: it's so it works with whatever table writer we're using, we can replace

File: cmd/pod/getPod.go (hunk 2 above)
Comment (DireLines) [reply cmd/pod/getPod.go]: no

**Review by DireLines — APPROVED** [review]
took a while to get to this because we had to implement the same struct for our own purposes, so needed to combine your version and ours. But after checking, it'll work fine. Thanks!

**Pull request merged**